.github/dependabot.yml

```diff
@@ -0,0 +1,156 @@
+###
```

Comment:
Thank you for the comments about the license boilerplate. I'll probably clean them up in a future commit.